organization/_posts/2020/2020-03-12-coordination.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "HSF Weekly Meeting #182, 12 March, 2020"
+layout: meetings
+---
+
+# {{page.title}}
+
+Present/Contributors: Graeme Stewart, Pere Mato, Eduardo Rodrigues, Ben Morgan, Paul Laycock, Attila Krasznahorkay, Mason Proffitt, Efe Yazgan, Daniel Elvira, Michel Jouvin, Kyle Knoepfel, Teng Jian Khoo, Liz Sexton-Kennedy, Maria Girone, Philippe Canal, Stefan Roiser, Daniel Elvira, Witek Pokorski, Andrea Rizzi, Chris Jones, David Lange, Gloria Corti, Horst Severini, 
+
+Apologies: Caterina Doglioni, Agnieszka Dziurda
+
+## News, general matters
+
+### HEP-CCE Kick Off Meeting
+There was an [organizing meeting](<https://indico.fnal.gov/event/23262/other-view?view=standard>) for a new project organization of the CCE which is a collaboration of ANL, BNL, Fermilab,and LBNL participants funded to work on HEP's most pressing and uncovered topics.
+
+We've agreed that going forward all technical meetings should fully public in the same way that IRIS-HEP meetings are. It seems to me that this could be coordinated through the HSF sub-group conveners.
```

Comment:
```suggestion
We've agreed that going forward all technical meetings should be fully public in the same way that IRIS-HEP meetings are. It seems that this could be coordinated through the HSF sub-group conveners.
```